src/workspace-data/data-table/uri-viewer/UriViewer.js

```diff
@@ -106,7 +106,7 @@ export const UriViewer = _.flow(
     return h(Fragment, [
       p({ style: { marginBottom: '0.5rem', fontWeight: 'bold' } }, ['Terminal download command']),
       p({ style: { marginBottom: '0.5rem', fontWeight: 500 } }, ['Download to:']),
-      div({ marginBottom: '0.5rem' }, [
+      div({ marginbottom: '0.5rem' }, [
```

Comment:
this case-change looks like a typo; revert?

Reply:
This isn't a typo, if I do not change it to `marginbottom` the error below appears in the console.

<img width="1775" alt="Screenshot 2024-08-22 at 7 47 23 PM" src="https://github.com/user-attachments/assets/f14ec1ac-5507-4ab4-a681-5717fc73187d">

Reply:
ah - perhaps this should actually be
```
div({ style: { marginbottom: '0.5rem' } }, [
```
?

If you do a case-sensitive search across the codebase, this line is the only occurrence of `marginbottom` and there are 197 occurrences of `marginBottom`. Since this is a CSS property, it seems like it should be in a `style` block. I realize this is legacy code and you didn't write it originally, but since you're modifying it in this PR … 

I don't know if by making the syntax actually work (inside a `style` block) you'll be changing the layout of the page. Feel free to call this out of scope for the current PR.

Reply:
I have [left this out of the PR](https://github.com/DataBiosphere/terra-ui/pull/5029/commits/6c5f08843c7102b27ee2b0e4df43910dc9f58df2)!